E2E-tests/report_slack.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+slack_webhook_url=$SLACK_WEBHOOK_URL
+jira_url=$JIRA_URL
```

Comment:
we need to add these to repo secrets

Reply:
I already did that. If its not working let me know.